h2o-jetty-8/src/main/java/ai/h2o/jetty8/Jetty8HTTPD.java

```diff
@@ -172,7 +181,7 @@ public void handle(String target,
                        Request baseRequest,
                        HttpServletRequest request,
                        HttpServletResponse response) throws IOException, ServletException {
-      H2O.getJetty().handle1(target, baseRequest, request, response);
+//      H2O.getJetty().handle1(target, baseRequest, request, response);
```

Comment:
Will this get re-enabled before the final merge? Needed for customer-specific builds.

Reply:
Can you point me to more detailed information about this? The function, as it was, looked unused and unusable. If we really need it, I would like to put there some javadocs that convinces any by-walker that dropping is actually not the best thing, and how it is/can be used.

How do the customer-specific builds work? Where are some examples? I need to know more details so that I can decide if the `H2oServletContainer` interface should be enhanced to support this kind of extensions.

BTW, will we support this also for jetty 9, or just jetty 8 for compatibility?

Reply:
Yes, I will send you a private message.

The function is indeed used and usable ;) Please don't remove anything that appears unused without consulting someone.

Reply:
And of course - feel free to document in javadoc why it is needed (if it ended up being needed after your refactoring).

Needs to be supported on both Jetty 8 and Jetty 9. Example will make it clear.